plugins/stronghold/src/lib.rs

```diff
@@ -405,15 +408,35 @@ impl Builder {
         }
     }
 
+    /// Initializes a stronghold plugin with argon2 as a default kdf
+    pub fn init_and_build_with_argon2<R: Runtime>() -> TauriPlugin<R> {
```

Comment:
to reduce duplicated code, I'd actually name this as `pub fn with_argon2` that calls the `new` function with the appropriate password hash function.